send_to_dock/test/test_send_to_dock_node.cpp

```diff
@@ -0,0 +1,131 @@
+// Copyright 2025 Husarion Sp. z o.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <memory>
+
+#include <gtest/gtest.h>
+#include <rclcpp/rclcpp.hpp>
+#include <rclcpp_action/rclcpp_action.hpp>
+
+#include <nav2_msgs/action/dock_robot.hpp>
+
+#include "send_to_dock/send_to_dock_node.hpp"
+
+using DockRobot = nav2_msgs::action::DockRobot;
+using ClientGoalHandleDockRobot = rclcpp_action::ClientGoalHandle<DockRobot>;
+using ServerGoalHandleDockRobot = rclcpp_action::ServerGoalHandle<DockRobot>;
+using SetBoolSrv = std_srvs::srv::SetBool;
+
+class SendToDockNodeWrapper : public send_to_dock::SendToDockNode {
+public:
+  SendToDockNodeWrapper() {};
```

Comment:
Let's add name param to the `SendToDockNode` constructor and add the explicit constructor call:
 ```suggestion
   SendToDockNodeWrapper():  send_to_dock::SendToDockNode("test_send_to_dock") {};
```